README.md

```diff
@@ -0,0 +1,153 @@
+- Manifest values (i32) identify the actor type, to be parsed as the
+  `fvm_shared::actor::builtin::Type` enum:
+    - System = 1
+    - Init = 2
+    - Cron = 3
+    - Account = 4
+    - Power = 5
+    - Miner = 6
+    - Market = 7
+    - PaymentChannel = 8
+    - Multisig = 9
+    - Reward = 10
+    - VerifiedRegistry = 11
+
+The CARv1 is embedded as a byte slice at the root of the library, and exported
+under the `BUNDLE_CAR` public const, for easier consumption by Rust code.
+
+Precompiled actor bundles may also be provided as release binaries in this repo,
+if requested by implementors.
+
+## Instructions for client implementations
+
+### Obtaining an actors bundle
+
+There are two options:
+
+1. Building from source.
+2. Downloading the precompiled release bundle from GitHub.
+
+Instructions to build from source (option 1):
+
+1. Clone the repo.
+2. Check out the relevant branch or tag (see Versioning section below).
+3. `cargo build` from the workspace root.
+4. Copy the CARv1 file generated the location printed in this log line:
+    ```
+   warning: bundle=/path/to/repo/target/debug/build/filecoin_canonical_actors_bundle-aef13b28a60e195b/out/bundle/bundle.car
+   ```
+   All output is printed as a warning only due to limitations in the Cargo build
+   script mechanisms.
+
+Both options are compatible with automation via scripts or CI pipelines.
+
+### Integrating an actors bundle
+
+This part is implementation-specific. Options include:
+
+1. Embedding the bundle's CARv1 bytes into the distribution's binary.
+2. Downloading CARv1 files on start (with some form of checksumming for added security).
+
+### Loading and using the actors bundle with ref-fvm
+
+Once the implementation has validated the authenticity of the bundle, it is
+expected to do the following:
+
+1. Import the CARv1 into the blockstore.
+2. Retain the root CID in memory, indexed by network version.
+3. Feed the root CID to ref-fvm's Machine constructor, to tell ref-fvm which
+   CodeCID maps to which built-in actor.
+
+### Multiple network version support
+
+Because every network version may be backed by different actor code,
+implementations should be ready to load multiple actor bundles and index them
+by network version.
+
+When instantiating the ref-fvm Machine, both the network version and the
+corresponding Manifest root CID must be passed.
+
+## Versioning
+
+A fair question is how crate versioning relates to the protocol concept of
+`ActorVersion`. We adopt a policy similar to specs-actors:
+
+- Major number in crate version correlates with `ActorVersion`.
+- We generally don't use minor versions; these are always set to `0`.
+- We strive for round major crate versions to denote the definitive release for
+  a given network upgrade. However, due to the inability to predict certain
+  aspects of software engineering, this is not a hard rule and further releases
+  may be made by bumping the patch number.
```

Comment:
This wasn't a goal of specs-actors at all. Many releases were branches with multiple minor version tags and the one released to the network was not `0`. The major version bump doesn't signal the end of a development stream, but the beginning of the next one.

Reply:
Interesting. Looking at the specs-actors version history, v3 was the last major line that used minor versions. All subsequent versions hit the network as vN.0.M. I assumed this was an intentional decision since version strings like v4.1 would be rather confusing in a world where clients model the actor version as a single int (denoting the major version)?